docker-compose-production.yml

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
this is more a `docker-compose.ci.yml` than a production-oriented configuration, that would need secrets to be passed in and wouldn't run a local `postgres` container to keep all state

Reply:
I've added a ticket for writing a `docker-compose.ci.yml` properly